libc/src/__support/FPUtil/CMakeLists.txt

```
@@ -209,6 +209,17 @@ add_header_library(
     libc.src.__support.macros.properties.types
 )
 
+add_header_library(
+  comparison_operations
+  HDRS
+    ComparisonOperations.h
+  DEPENDS
+    .fenv_impl
+    .fp_bits
+  libc.src.__support.macros.config
+    libc.src.__support.CPP.type_traits
```

Nit: formatting and sorting.

```suggestion
  DEPENDS
    .fenv_impl
    .fp_bits
    libc.src.__support.CPP.type_traits
    libc.src.__support.macros.config
```